canonical-websites/www.ubuntu.com
# Upgrade to ./run v1.1.0

Upgrade to the soon-to-be-minted v1.1.0 of generator-canonical-webteam
(https://github.com/canonical-webteam/generator-canonical-webteam/pull/35).

QA
--

``` bash
./run  # Check the site runs properly
./run clean
./run
./run test  # many tests fail :(
./run help
./run watch
... etc.
```
